airflow/gcp/operators/bigquery.py

```diff
@@ -217,7 +217,17 @@ class BigQueryGetDataOperator(BaseOperator):
     be equal to the number of rows fetched. Each element in the list will again be a list
     where element would represent the columns values for that row.
 
```

Comment:
This description seems to be not valid anymore. If someone is checking the number of the elements of returned list (and it is dictionary) it will be always three.